corehq/apps/data_cleaning/exceptions.py

```diff
@@ -1,2 +1,6 @@
 class UnsupportedActionException(Exception):
     """Raised when an unknown action is encountered"""
+
+
+class UnsupportedFilterValueException(Exception):
+    """Raised when a BulkEditColumnFitler"""
```

Comment:
Sorry, when is this raised?

Reply:
updated here https://github.com/dimagi/commcare-hq/pull/35846/commits/312c2cd1d3ba6d6e4ec7f313aed7c901a2980f7a